src/js/base/module/AutoReplace.js

```diff
@@ -0,0 +1,75 @@
+import $ from 'jquery';
```

Comment:
How about to add a credit of @dubiousandrew who creates `AutoReplace` module at first?